distribution.pm

```diff
@@ -141,8 +143,10 @@ sub script_run {
     }
     testapi::type_string "$cmd";
     if ($args{timeout} > 0) {
+        my $terminator = ((substr $cmd, -1) =~ qr/(?<!\\)&/) ? '' : ';';
```

Comment:
I don't think the `(?<!\\)` will have any effect combined with `substr $cmd, -1`. Did you test it?

Reply:
oh dammit, I tested the regex, yes. But not the substr of course.

Reply:
Fixed